indirect_value_test.cpp

```diff
@@ -39,17 +39,16 @@ void SelfAssign(T& t, U&& u) {
 
 TEST_CASE("Ensure that indirect_value uses the minimum space requirements",
           "[indirect_value.sizeof]") {
-  STATIC_REQUIRE(sizeof(indirect_value<int>) ==
-                 sizeof(std::unique_ptr<int>));
+  STATIC_REQUIRE(sizeof(indirect_value<int>) == sizeof(std::unique_ptr<int>));
```

Comment:
Comparing against `std::unique_ptr` here is no longer the right choice since indirect_value is no longer implemented in terms of `std::unique_ptr`.  Comparing against a raw pointer probably makes more sense now.